src/DssSpell.t.sol

```diff
@@ -1338,25 +1563,56 @@ contract DssSpellTest is DSTest, DSMath {
         checkCollateralValues(afterSpell);
     }
 
-    // function testChainlogValues() public {
-    //     vote(address(spell));
-    //     scheduleWaitAndCast(address(spell));
-    //     assertTrue(spell.done());
+    function testChainlogValues() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
+
+        ChainlogAbstract chainLog = ChainlogAbstract(addr.addr("CHANGELOG"));
+
+        assertEq(
+            chainLog.getAddress("RWA002"),
+            addr.addr("RWA002")
+        );
+        assertEq(
+            chainLog.getAddress("MCD_JOIN_RWA002_A"),
+            addr.addr("MCD_JOIN_RWA002_A")
+        );
+        assertEq(
+            chainLog.getAddress("RWA002_A_URN"),
+            addr.addr("RWA002_A_URN")
+        );
+        assertEq(
+            chainLog.getAddress("RWA002_A_INPUT_CONDUIT"),
+            addr.addr("RWA002_A_INPUT_CONDUIT")
+        );
+        assertEq(
+            chainLog.getAddress("RWA002_A_OUTPUT_CONDUIT"),
+            addr.addr("RWA002_A_OUTPUT_CONDUIT")
+        );
+        assertEq(
+            chainLog.getAddress("MIP21_LIQUIDATION_ORACLE"),
+            addr.addr("MIP21_LIQUIDATION_ORACLE")
+        );
 
-    //     ChainlogAbstract chainLog = ChainlogAbstract(addr.addr("CHANGELOG"));
+        (,address pip,,) = RwaLiquidationLike(
+            addr.addr("MIP21_LIQUIDATION_ORACLE")
+        ).ilks(ilk);
+        assertTrue(chainLog.getAddress("PIP_RWA002") == pip);
 
-    // }
+        assertEq(chainLog.version(), "1.2.11");
+    }
 
-    // function testCollateralIntegrations() public {
-    //     vote(address(spell));
-    //     spell.schedule();
-    //     castPreviousSpell();
-    //     hevm.warp(spell.nextCastTime());
-    //     spell.cast();
-    //     assertTrue(spell.done());
+    function testCollateralIntegrations() public {
+        vote(address(spell));
+        spell.schedule();
+        castPreviousSpell();
+        hevm.warp(spell.nextCastTime());
+        spell.cast();
+        assertTrue(spell.done());
 
-    //     // Insert new collateral tests here
-    // }
+        // Insert new collateral tests here
```

Comment:
Missing tests?

Reply:
no, this is just a placeholder for standard collateral tests.  I just uncommented this section because... why not.